### PR TITLE
JW-1074 : Auto generate contrail-go-api types

### PR DESCRIPTION
**Pull request by kaweue**

### CODE REVIEW

**Comment by kaweue** [comment]
Retest this please

**Comment by kaweue** [comment]
Retest this please

**Comment by kaweue** [comment]
Retest this please

**Review by jacek-izykowski — DISMISSED** [review]

File: CIScripts/BuildFunctions.ps1 [hunk 1]
```diff
@@ -84,6 +84,10 @@ function Invoke-DockerDriverBuild {
     $Env:GOPATH=pwd
     $srcPath = "$Env:GOPATH/src/$DriverSrcPath"
 
+    $Job.Step("Contrail-go-api source code generation", {
+        python tools/generateds/generateDS.py -f -o $srcPath/vendor/github.com/Juniper/contrail-go-api/types/ -g golang-api controller/src/schema/vnc_cfg.xsd
+    })
+
```
Comment (mclapinski) [inline CIScripts/BuildFunctions.ps1]: spaces

**Review by mclapinski — CHANGES_REQUESTED** [review]

**Review by mclapinski — APPROVED** [review]

**Pull request merged**